courses/fundamentals_of_ada/070_subprograms.rst

```diff
@@ -381,14 +385,20 @@ Actual Parameters Respect Constraints
      Foo (Q); -- runtime error if Q <= 0
      Foo (P);
 
------------------
-Parameter Modes
------------------
+---------------------------
+Parameter and Return Mode
```

Comment:
No reference to "Return" on this page
If you add one, it should be "Parameter Modes and Return"

Reply:
The last bullet is referencing return